cockpit/src/index.html

```diff
@@ -25,6 +25,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <link rel="stylesheet" href="../base1/patternfly.css">
+    <link rel="stylesheet" href="../base1/patternfly-additions.css">
```

Comment:
This is probably duplication of import from subscriptions.css

Reply:
The `../base1/patternfly.css` is part of default cockpit installation, but it doesn't include all we need for ListView. The `../base1/patternfly-additions.css` probably will by part of next cockpit release.

Importing `patternfly.css` in `subscription.css` wasn't good idea, because it adds many unused files to dist (and rpm package). I will import only `patternfly-additions.css` in `subscription.css` to minimize the result size of RPM. Importing of `patternfly-additions.css` could be possibly removed in the future.